instances/treasury-factory.near/widget/components/create-treasury/SummaryStep.jsx

```diff
@@ -0,0 +1,368 @@
+const { formFields } = props;
+
+const REQUIRED_BALANCE = 12;
+
+const [showCongratsModal, setShowCongratsModal] = useState(false);
+
+const Section = styled.div`
+  display: flex;
+  flex-direction: column;
+  padding: 10px 0;
+  border-bottom: ${(props) => (props.withBorder ? "1px solid #E2E6EC" : "0")};
+
+  label {
+    font-size: 12px;
+    color: #999999;
+    margin-bottom: 5px;
+  }
+
+  i {
+    font-size: 18px;
+    color: #060606;
+  }
+
+  ul {
+    margin: 0;
+    padding: 0;
+
+    li {
+      list-style: none;
+      padding: 8px 0;
+      border-bottom: 1px solid #e2e6ec;
+
+      &:last-child {
+        border: 0;
+      }
+    }
+  }
+`;
+
+const Badge = styled.div`
+  border: 1px solid #e2e6ec;
+  border-radius: 32px;
+  padding: 4px 10px;
+`;
+
+const Item = styled.div`
+  border-bottom: 1px solid #e2e6ec;
+  padding: 10px 0;
+
+  &:last-child {
+    border: 0;
+  }
+`;
+
+const WidgetItemLink = styled.div`
+  border-bottom: 1px solid #e2e6ec;
+  padding: 10px 0;
+
+  &:last-child {
+    border: 0;
+  }
+
+  small {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 15px;
+    color: #b3b3b3;
+  }
+
+  span {
+    font-size: 14px;
+  }
+
+  a {
+    color: #060606;
+  }
+`;
+
+const PERMISSIONS = {
+  create: "Create",
+  edit: "Edit",
+  vote: "Vote",
+};
+
+const storageAccountName = Storage.get("accountName");
+
+useEffect(() => {
+  if (storageAccountName) {
+    setShowCongratsModal(true);
+    Storage.set("accountName", null);
+  }
+}, [storageAccountName]);
+
```

Comment:
why are we using local storage?
and would it matter if it is changed by some other [widget](https://docs.near.org/build/near-components/anatomy/web-methods#storageset)
may be we should be private storage?